Libraries/LibJS/StringPrototype.cpp

```diff
@@ -48,7 +48,7 @@ StringPrototype::StringPrototype()
         ASSERT(this_object);
         i32 index = 0;
         if (!arguments.is_empty())
-            index = arguments[0].to_i32();
+            index = static_cast<i32>(arguments[0].to_number().as_double());
```

Comment:
I don't want to have to do this every time I want to convert a `JS::Value` to an `i32`.
Let's have a helper for it.